app/src/main/res/values-en/strings.xml

```diff
@@ -1258,25 +1258,25 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_smoothness_description_good_sett">Unusually flat stones, regular pattern, very shallow or narrow gaps</string>
     <string name="quest_smoothness_description_good_compacted_gravel">Hard-packed, very little loose gravel or sand</string>
     <string name="quest_smoothness_description_intermediate_paved">Unusually rough surface, wider gaps, cracks, shallow ruts, potholes or other damage</string>
-    <string name="quest_smoothness_description_intermediate_paving_stones">Wider gaps, possibly some displaced stones or other damage</string>
+    <string name="quest_smoothness_description_intermediate_paving_stones">Wider gaps, possibly some loose stones or other damage</string>
     <string name="quest_smoothness_description_intermediate_sett">Flat stones or with square edges, shallow or narrow gaps</string>
     <string name="quest_smoothness_description_intermediate_compacted_gravel">Fine gravel, shallow bumps or potholes</string>
     <string name="quest_smoothness_description_bad_paved">Big gaps, potholes, ruts, crumbled paving or other damage</string>
-    <string name="quest_smoothness_description_bad_paving_stones">Displaced stones, big gaps or other damage</string>
+    <string name="quest_smoothness_description_bad_paving_stones">Loose stones, big gaps or other damage</string>
     <string name="quest_smoothness_description_bad_sett">Weathered stone or irregular shapes, big gaps or damage</string>
     <string name="quest_smoothness_description_bad_compacted_gravel">Some bigger gravel, bumps, ruts or potholes</string>
-    <string name="quest_smoothness_description_very_bad_paved">Deep potholes, ruts or other dangerous damage</string>
-    <string name="quest_smoothness_description_very_bad_sett">Big gaps, displaced, missing or irregularly shaped stones or dangerous damage</string>
+    <string name="quest_smoothness_description_very_bad_paved">Deep potholes, ruts or other hazardous damage</string>
+    <string name="quest_smoothness_description_very_bad_sett">Big gaps, loose, missing or irregularly shaped stones or hazardous damage</string>
```

Comment:
I know this text fits to the current picture better because it shows stones that look loose. But actually, the picture is not that great and we are looking for a replacement. We really mean "displaced" here, as in the regular pattern of stones is broken up by stones rotated a bit (around any axis), the gaps between the stones are irregular etc.

Reply:
Restored original text